pyproject.toml

```diff
@@ -10,4 +10,8 @@ requires = [
     "epicscorelibs>=7.0.7.99.1.1a2; python_version>='3.12'",
     "pvxslibs>=1.3.1",
     "pvxslibs>=1.3.2a1; python_version>='3.12'",
+    "packaging>=24.2",
 ]
+
+[project]
+license-files = ["LICENSE"]
```

Comment:
This appears to be the cause of the CI failure as it seems as soon as you include the `project` header there are some required keys.